### PR TITLE
Add more installation information about Fedora & Co + Raspbian

### LINKED ISSUE #316: Unable to install on Raspbian (librsync.h/librsync.c)
Manual install on Ubuntu via pip3: no problem.  But I am unable to do the same on a raspberry pi.

``python-librsync`` is installed, but it appears to be looking for the C source code for librsync, and I am not sure how to install that.

Truncated log:
```
>sudo pip3 install rdiff-backup
Looking in indexes: https://pypi.org/simple, https://www.piwheels.org/simple
Collecting rdiff-backup
  Using cached rdiff-backup-2.0.0.tar.gz (408 kB)
Building wheels for collected packages: rdiff-backup
  ERROR: Command errored out with exit status 1:
   arm-linux-gnueabihf-gcc -pthread -DNDEBUG -g -fwrapv -O2 -Wall -g -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -fPIC -I/usr/include/python3.7m -c src/_librsyncmodule.c -o build/temp.linux-armv7l-3.7/src/_librsyncmodule.o
  src/_librsyncmodule.c:25:10: fatal error: librsync.h: No such file or directory
   #include <librsync.h>
            ^~~~~~~~~~~~
```
How does one install the required C librsync library on Raspbian?

--

Also, just to avoid traps, for beginners, under installation the line:
``curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py sudo python3 get-pip.py sudo pip3 install rdiff-backup``
Should perhaps read:
``curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py && sudo python3 get-pip.py && sudo pip3 install rdiff-backup``

### PR DESCRIPTION
**Pull request by ericzolf**
NEW: add Fedora and RHEL to installation instructions, and evoke Raspbian, closes #316

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -171,6 +190,24 @@ sudo apt update
 sudo apt install rdiff-backup
 ```
 
+### Fedora, CentOS and RHEL
```
Comment (ikus060) [inline README.md:193]: Are you sure the "level" are the irght one here ? I would have put "####"

File: README.md [hunk 2]
```diff
@@ -55,30 +55,49 @@ sudo apt update
...
+sudo yum install dnf-plugins-core epel-release
+sudo dnf copr enable frankcrawford/rdiff-backup
+sudo yum install rdiff-backup
 ```
 
-### Debian (From PyPi)
+### Fedora 32+
+
+```
+sudo dnf install rdiff-backup
```
Comment (ikus060) [inline README.md:77]: More a question. Fedora 32+ already have v2.0.0 in default repo ? I tought rdiff-backup would need epel...
If we need epel, we should add the command line to install epel.

Comment (ericzolf) [reply README.md:77]: EPEL is only for CentOS and RHEL, Fedora has much more packages and doesn't need it. And, yes, it's there, thanks to Frank!

File: README.md (hunk 1 above)
Comment (ericzolf) [reply README.md:193]: Well spotted and fixed.

**Pull request merged**